docs/starlark/experimental/go.md

```diff
@@ -0,0 +1,26 @@
+
+# Go
+
+Provides methods for building and testing Go modules.
+
+Import URL: `github.com/mesosphere/dispatch-catalog/starlark/stable/go`
```

Comment:
Might be slightly easier to use to have this as:

Import the module at the top of your Dispatchfile:
\```
import "github.com/mesosphere/dispatch-catalog/starlark/go@master";
\```
etc...

Reply:
Sounds good!